unique_toolkit/unique_toolkit/content/service.py

```diff
@@ -18,21 +18,29 @@
 )
 
 
-class ContentService:
-    DEFAULT_SEARCH_LANGUAGE = "english"
+class ContentService(BaseService):
+    """
+    Provides methods for searching, downloading and uploading content in the knowledge base.
+
+    Attributes:
+        state (ChatState): The chat state.
+        logger (Optional[logging.Logger]): The logger. Defaults to None.
+    """
 
     def __init__(self, state: ChatState, logger: Optional[logging.Logger] = None):
-        self.state = state
-        self.logger = logger or logging.getLogger(__name__)
+        super().__init__(state, logger)
+
+    DEFAULT_SEARCH_LANGUAGE = "english"
 
     def search_content_chunks(
         self,
         search_string: str,
         search_type: ContentSearchType,
         limit: int,
-        reranker_config: Optional[RerankerConfig] = None,
         search_language: str = DEFAULT_SEARCH_LANGUAGE,
-        scope_ids: Optional[list[str]] = None,
+        reranker_config: Optional[RerankerConfig] = None,
+        scope_ids: Optional[list[str]] = [],
```

Comment:
Default value here should be `None`. 
It is also not a good idea to assign a `[]` and `{}` as default in methods. (https://www.pullrequest.com/blog/python-pitfalls-the-perils-of-using-lists-and-dicts-as-default-arguments/)